spec/acceptance/03_aptitude_spec.rb

```diff
@@ -40,7 +40,7 @@ def set_package_state host, package, state
     hosts.each do |host|
       localpath = File.dirname(__FILE__) + '/fixtures'
       remotepath = '/usr/local/localrepo'
-      on host, "mkdir #{remotepath}"
+      on host, "mkdir #{remotepath}", :accept_all_exit_codes => true
```

Comment:
To avoid `EEXIST`. Prior to this PR we had a single `mkdir #{remotepath}` among all the acceptance tests. Now there are two...